### PR TITLE
BUGFIX: FET-942 Update some packages to fix sonar vulnerabilities

### PR DESCRIPTION
**Pull request by fiedlr**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: package.json [hunk 1]
```diff
@@ -33,18 +33,18 @@
...
     "express": "^4.17.1",
     "express-redirect": "^1.2.2",
     "http-proxy": "^1.18.1",
     "lodash.defaults": "^4.2.0",
-    "yargs": "^14.2.0"
+    "yargs": "^17.3.0"
   },
   "devDependencies": {
-    "eslint": "^6.8.0",
+    "eslint": "7.32.0",
```
Comment (pbenes) [inline package.json:44]: Is there a reason for having non-caret version here? If possible, use "^7.32.0" (use `yarn upgrade eslint@^7.32.0`)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by pbenes — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=grunt-grizzly-gate-PR72

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/grunt-grizzly-merge-compliance-check/18/ : SUCCESS in 20s
- https://checklist.intgdc.com/job/compliance/job/grunt-grizzly-sonarqube-scan-zuul-gate/4/ : SUCCESS in 35s
- https://checklist.intgdc.com/job/compliance/job/grunt-grizzly-fossa-scanning-zuul-el7/14/ : SUCCESS in 55s (non-voting)

**Pull request merged**